res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -858,6 +859,7 @@ NumarkMixtrack3.deck = function(decknum) {
     // NMTP3 is a "Model A" controller for scratching, it centers on 0.
     // See http://www.mixxx.org/wiki/doku.php/midi_scripting#scratching
     // and see "Jogger" object constructor
+    this.beatJumpSize = 1;
```

Comment:
FYI, I just tried to install Mixxx 2.1 on Windows 10 and the installer failed. Also, I got 3 different warnings asking to confirm that I wanted to install the package; the first one saying that Windows did not recommend installing it, likely because package was not signed.